features/mouse-events.yml

```diff
@@ -0,0 +1,52 @@
+name: Mouse events
+description: Mouse events, such as `click`, `mousedown`, or `mousemove`, fire when users interact with a pointing device such a mouse or trackpad.
```

Comment:
For site compat reasons the events also fire when interacting with touch screens, although in that case touch events are also fired. And pointer events. I'm not sure how much to say, WDYT?